pkg/log/log.go

```diff
@@ -0,0 +1,70 @@
+package log
```

Comment:
Is it enough to use kitex's klog here?

Reply:
Please reply with an answer/reason before resolve a conversation

Reply:
Okay, using kitex's klog should not be enough here, because it also needs to output a success or error hint message